roles/common/templates/ubuntu.list

```diff
@@ -0,0 +1,9 @@
+# Ubuntu Main Archive
```

Comment:
Those  are the default repositories, and is not a template format.
Do we really need this?